pom.xml

```diff
@@ -40,7 +40,7 @@
       <dependency>
         <groupId>org.georchestra</groupId>
         <artifactId>georchestra-ldap-account-management</artifactId>
-        <version>23.1-SNAPSHOT</version>
+	<version>${project.version}</version>
```

Comment:
so you stick the gateway with the georchestra version ?

Reply:
kind of makes sense? dunno, for the time being I didn't want to think to hard about it

Reply:
yes, it was just a remark while reviewing, it sounds ok to me